src/Control/Monad/Eff/Random.purs

```diff
@@ -28,7 +28,18 @@ foreign import random :: forall e. Eff (random :: RANDOM | e) Number
 randomInt :: forall e. Int -> Int -> Eff (random :: RANDOM | e) Int
 randomInt low high = do
   n <- random
-  pure <<< U.fromJust <<< fromNumber <<< Math.floor $ toNumber (high - low + one) * n + toNumber low
+  let asNumber = (toNumber high - toNumber low + one) * n + toNumber low
+  return $ safeFromNumber asNumber
+
+-- | Safely convert a Number into an Int, by first flooring and then clamping
+-- to the valid integer range.
+safeFromNumber :: Number -> Int
```

Comment:
No, it shouldn't, but I didn't do that yet becaues I thought you might want to put this function into `integers` instead. I can imagine it being useful in other contexts.